synapse/models/orgs.py

```diff
@@ -10,6 +10,17 @@
     'partnership',
 )
 
+dealstatus = (
+    (0, 'new'),
+    (0, 'research'),
+    (10, 'pitching'),
+    (20, 'offer made'),
+    (50, 'negotiation'),
+    (50, 'committed'),
+    (60, 'delivered'),
+    (70, 'done'),
```

Comment:
is stalled or cancelled/dead also a valid status?

Reply:
Yea, def need to talk through these defaults with shevenom and maybe some outsiders :D